lib/code_corps/github/event/issues/validator.ex

```diff
@@ -0,0 +1,20 @@
+defmodule CodeCorps.GitHub.Event.Issues.Validator do
+  @moduledoc ~S"""
+  In charge of validatng a GitHub Issue webhook payload.
+
+  https://developer.github.com/v3/activity/events/types/#issuesevent
+  """
+
+  @doc ~S"""
+  Returns `true` if all keys required to properly handle an Issue webhook are
+  present in the provided payload.
+  """
+  @spec valid?(map) :: boolean
+  def valid?(%{
+    "issue" => %{
+      "id" => _, "title" => _, "body" => _, "state" => _,
+      "user" => %{"id" => _}
+    },
+    "repository" => %{"id" => _}}), do: true
```

Comment:
This function returns true if the payload possesses the minimum required data to handle the `Issues` webhook.

Basically this is our "first line of defense", to make sure nothing has changed with the Github API spec and the structure of the payload is as we expect it to be.